Backend/app/dashboard/Lib/src/Storage/Queries/Person.hs

```diff
@@ -58,11 +59,12 @@ findByEmailAndPassword ::
   m (Maybe Person)
 findByEmailAndPassword email password = do
   emailDbHash <- getDbHash email
+  emailDbHashLower <- getDbHash (T.toLower email)
   passwordDbHash <- getDbHash password
   findOne $ do
     person <- from $ table @PersonT
     where_ $
-      person ^. PersonEmailHash ==. val emailDbHash
+      (person ^. PersonEmailHash ==. val emailDbHash ||. person ^. PersonEmailHash ==. val emailDbHashLower)
```

Comment:
You can make table's hash lower too and compare. Also, I doubt if hash is case insensitive and if it isn't, you should lowercase the email before fetching it and dumping to table too

Reply:
> You can make table's hash lower too and compare instead of OR condition. Also, I doubt that hash is case insensitive and if it isn't, you should lowercase the email before fetching and dumping to table too

How? It is hash, not the actual Email, you can't get hash of lower cased email out of whatever cased one.
It would be better to have all emails in lower case in db, but idk how to migrate it if it is not already the case...

Reply:
Yes, I'm saying that when a person is created, lower its email - take hash - and then dump to db.
Then while comparing - lower the email - take hash and compare without lowering anything else.

Reply:
@0utkarsh ,I also thought the same , but max users are already created what to do with that ?

Reply:
I think the number of users should be very less rn, so we can create an API to update email which only admin will have access to. Admin can manually update such person's email.

Also, since external people are going to use dashboard, we should do this soon. If we are not doing this soon, at least make sure future users don't face this issue by making the necessary change in `createPerson` API at least.
cc: @KashishKhandelwal @hkmangla

Reply:
Make sense. @saurav2401 Lets do one thing, in dashboard create person API, make email id always lower before creating entry in DB. Here just make this `emailDbHash <- getDbHash email` to this `emailDbHash <- getDbHash (T.toLower email)). Don't change anwhere else. In DB,  I will make already created email lower case manually.

Reply:
ok @hkmangla